enos/modules/target_ec2_fleet/main.tf

```diff
@@ -336,3 +336,13 @@ data "aws_instance" "targets" {
 
   instance_id = aws_ec2_fleet.targets.fleet_instance_set[0].instance_ids[each.key]
 }
+
+module "disable_selinux" {
+  source = "../disable_selinux"
+  count  = var.disable_selinux == true ? 1 : 0
+
+  hosts = { for idx in range(var.instance_count) : idx => {
+    public_ip  = aws_instance.targets[idx].public_ip
```

Comment:
Recording here that we discovered some issues with the target_ec2_fleet module so will skip adding disable_selinux to this and the target_ec2_spot_fleet module for now, since we are not actively using either of those.